database/src/main/postgres/runs/partitionings.ddl

```diff
@@ -13,18 +13,16 @@
  * limitations under the License.
  */
 
--- DROP TABLE IF EXISTS runs.segmentations;
-
-CREATE TABLE runs.segmentations
+CREATE TABLE runs.partitionings
 (
-    id_segmentation         BIGINT NOT NULL DEFAULT global_id(),
-    segmentation            HSTORE NOT NULL,
+    id_partitioning         BIGINT NOT NULL DEFAULT global_id(),
+    partitioning            JSONB NOT NULL, -- TODO add  partitioning validity check #69
     created_by              TEXT NOT NULL,
     created_at              TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
-    CONSTRAINT segmentations_pk PRIMARY KEY (id_segmentation)
+    CONSTRAINT segmentations_pk PRIMARY KEY (id_partitioning)
 );
 
-ALTER TABLE runs.segmentations
-    ADD CONSTRAINT segmentations_unq UNIQUE (segmentation);
+ALTER TABLE runs.partitionings
+    ADD CONSTRAINT segmentations_unq UNIQUE (partitioning);
```

Comment:
The constraint might also be renamed to partitioning_unq.

Reply:
We might want to consider indexing the partitioning JSONB column as it's being used in _get_id_partitioning function when looking up partitioning id.

Reply:
Unique constraint automatically means an index created.